README.md

```diff
@@ -52,6 +52,9 @@ requirement. By default the published versions of those projects are used
 ## Run it locally
 
 ```bash
+# Create a link to ink! in the local examples of the `ink-waterfall`.
+ln -s /path/to/ink/ ./examples/ink
```

Comment:
Why not just use a released version of ink? The CI will replace it with current master anyways.

Reply:
This line:
`find ./examples/ -name 'Cargo.toml' -exec sed -i "s|../ink|../$INK_EXAMPLES_PATH|g" {} \;`

But I see why now. We want to run it against a custom version locally, too.